apps/ledger-live-mobile/src/screens/Settings/Debug/Generators/ImportBridgeStreamData.tsx

```diff
@@ -11,25 +12,52 @@ type Props = {
 };
 export default function ImportBridgeStreamData({ title, dataStr }: Props) {
   const navigation = useNavigation();
-  return (
-    <SettingsRow
-      title={title}
-      desc={dataStr ? "Import string detected" : "No import string detected"}
-      iconLeft={<Icons.ImportMedium size={24} color="black" />}
-      onPress={() => {
-        if (!dataStr) return;
 
-        const data = JSON.parse(
-          Buffer.from(dataStr, "base64").toString("utf8"),
-        );
+  const importString = useCallback(
+    (dataStr: Parameters<typeof Buffer.from>[0]) => {
```

Comment:
why not just set the type to `string` ?

Reply:
It seemed necessary at some point when I was reworking this component but yeah it's not actually 😅 thanks!